app/distros/octane.js

```diff
@@ -0,0 +1,91 @@
+var request = require('request');
+
+function init() {
+  var module = {
+    id: 'octane',
+    profile: 'lightning'
+  };
+
+  module.option = {
+    name: 'Octane',
+    value: module.id
+  };
+
+  module.versions = [
+    {name: 'Octane 8.x', value: '8.x'}
+  ];
+
+  module.versionDefault = '8.x';
+
+  module.description = 'This project is built on [' + module.option.name + '] distribution.';
+
+  module.releaseVersion = function(majorVersion, done, cb) {
+    cb(null, '8.x', done);
+  };
+
+  module.drushMakeFile = function(yo, options, done) {
+    done();
+  };
+
+  // Merge keys from jsonB into jsonA.
+  // Existing values of jsonA are overwritten if they appear in jsonB.
+  // If recurse is true, call recursively for one level of subkeys.
+  function mergeJson(jsonA, jsonB, recurse) {
```

Comment:
This is just complicated enough to ask, should we be using https://www.npmjs.com/package/merge or similar?